bedcov.c

```diff
@@ -188,13 +217,21 @@ int main_bedcov(int argc, char *argv[])
                             if (pi->is_del || pi->is_refskip) ++m;
                         }
                     }
-                    cnt[i] += n_plp[i] - m;
+                    int pd = n_plp[i] - m;
+                    cnt[i] += pd;
+                    if (min_depth >= 0 && pd > min_depth) pcov[i]++;
```

Comment:
Should this be `pd >= min_depth`?  Help message says "Number of bases with coverage above and including this value".

Reply:
I see that mpileup returns a reference skip, by default. I might have been misled by my test data.